levels/inside_perimeter/objectives/neumann_key/validator.js

```diff
@@ -1,34 +1,61 @@
-module.exports = async (helper) => {
-  const answer1 = helper.getNormalizedInput('answer1');
-  const answer2 = helper.getNormalizedInput('answer2');
-  const answer3 = helper.getNormalizedInput('answer3');
+function isValidAnswer1Option(answer1, answer1OptionsMap) {
+  return Object.keys(answer1OptionsMap).some(
+    (option) =>
+      option === answer1 || answer1OptionsMap[option].toLowerCase() === answer1
+  );
+}
 
+module.exports = async (helper) => {
+  const answer1 = helper.getNormalizedInput("answer1");
+  const answer2 = helper.getNormalizedInput("answer2");
+  const answer3 = helper.getNormalizedInput("answer3");
+  const answer1OptionsMap = {
+    a: "GET",
+    b: "POST",
+    c: "UPDATE",
+    d: "DELETE",
+  };
 
   if (!answer1 || !answer2 || !answer3) {
     return helper.fail(`
       Please answer all questions!
     `);
   }
 
-  if (answer1 !== 'c') {
+  if (!isValidAnswer1Option(answer1, answer1OptionsMap)) {
+    return helper.fail(
+      `"${answer1}" is not a valid answer! Please choose between "a", "b", "c", and "d"!`
+    );
+  }
+
+  if (answer1 !== "c" && answer1 !== "update") {
+    // Because of the "isValidAnswer1Option" check, we know that the player either entered a single letter answer
+    // or one of the labels (i.e. GET, POST, UPDATE, DELETE). If the player typed in a letter, it'll get the correct
+    // verb mapping, otherwise it'll uppercase the input.
     return helper.fail(`
-      Uh oh, the first challenge question is incorrect, ${answer1.toUpperCase()} *is* an HTTP Method. Visit the [MDN documentation to learn more about HTTP methods](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods) or check out the information provided here in the hack interface.
+      Uh oh, the first challenge question is incorrect, "${
+        answer1OptionsMap[answer1] || answer1.toUpperCase()
+      }" *is* an HTTP Method. Visit the [MDN documentation to learn more about HTTP methods](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods) or check out the information provided here in the hack interface.
```

Comment:
Markdown does not work in objective validation error messages. I think this needs to be an HTML `<a>` tag instead

Reply:
Fixed!